components/NewsReply.vue

```diff
@@ -452,12 +452,11 @@ export default {
     showInfo() {
       this.showProfileModal = true
     },
-    async replyReply() {
+    replyReply() {
       console.log('Replying to', this.replyid, this.reply)
       this.replyingTo = this.replyid
       this.showReplyBox = true
 
-      await this.waitForRef('replybox')
       this.$refs.replybox.$el.focus()
```

Comment:
`nextTick`?

Reply:
I don't think this one is needed. It works just fine without any additional waiting (tested manually).

Reply:
I think you may have tested the case of the Reply button for the top of the thread rather than replying to a reply.  I've tested replying to a reply and it does fail.

https://gyazo.com/a7f863bbd83aa5bd95bbcbe54ed7a14c

I've pushed this change so that we can get this PR released.